product-scenarios/5-route-messages-between-systems/5.1-Route-based-on-the-content-of-the-messages/5.1.1-Route-messages-based-on-the-given-xpath/5.1.1.1-Using-switch-mediator-on-given-xpath/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we add correct license header? Year should be 2019. Shall we update other files as well?

Reply:
Done